app/javascript/react/components/AddStatsForm/AddStatsForm.js

```diff
@@ -1,7 +1,7 @@
+/* eslint-disable react/no-array-index-key */
 import React, { useState, useEffect } from "react";
 import PropTypes from "prop-types";
 import { gql, useQuery } from "@apollo/client";
-import uuid from "react-uuid";
```

Comment:
cool - `make build` would have broken on this (Not sure if we run the `yarn lint` or what have you in Circle CI) - ultimately we should change Circle CI to just run the `make build` command